docs/hip-7-changes.md

```diff
@@ -0,0 +1,312 @@
+# CUDA compatibility changes to HIP API
+
+To improve code portability between AMD and NVIDIA GPU programming models specific changes were made to the HIP API in the 7.0 release to simplify cross-platform programming. These changes align HIP C++ even more closely with NVIDIA CUDA. These changes are incompatible with prior releases, and may require recompiling existing HIP applications for use in the 7.0 release, or editing and recompiling code in some cases. Best case, the change requires no modification of existing applications. These changes were made available in a preview release based on the 6.4.1 release, and as such hopefully you have had advanced notice and prepared for the following changes.
+
+## Changes in the HIP Runtime API
+
+### hipGetLastError
+
+Before ROCm 7.0, `hipGetLastError` was not fully compliant with CUDA's behavior. The purpose of this change is to get the error code returned by `hipGetLastError` to be the last actual error caught in the current thread during the application execution. Neither `hipSuccess` nor `hipErrorNotReady` is considered an error. Take the following code as an example:
```

Comment:
Use cpp func for the HIP functions. Also we can link the error handling page: https://rocm.docs.amd.com/projects/HIP/en/latest/how-to/hip_runtime_api/error_handling.html